packages/cli/package.json

```diff
@@ -33,6 +33,7 @@
     "nsp": "^3.1.0",
     "rimraf": "^2.6.2",
     "sinon": "^4.1.2",
+    "source-map-support": "^0.5.2",
```

Comment:
Same as above

Reply:
And same reply as above applies here too 😄